src/utilities/piral-jest-utils/package.json

```diff
@@ -41,13 +41,13 @@
     "@wojtekmaj/enzyme-adapter-react-17": "^0.3.2",
     "enzyme": "^3.11.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^25.0.0",
-    "jest-cli": "^25.0.0",
-    "ts-jest": "^25.0.0"
+    "jest": "^26.6.3",
```

Comment:
We could go here for the `^26.0.0` versions. Reason: It will still resolve to the latest 26 in our case, but if somebody has, e.g., `26.1.0` as fixed dependency to `jest` it would not install 2 dependencies, but just one.